src/main/java/frc/utils/RoborioUtils.java

```diff
@@ -0,0 +1,29 @@
+package frc.utils;
+
+import edu.wpi.first.wpilibj.RobotController;
+import edu.wpi.first.wpilibj.Timer;
+
+public class RoborioUtils {
+
+    private static double lastTime;
+
+    private static double currentTime;
+
+    public static void updateCurrentCycleTime() {
+        lastTime = currentTime;
+        currentTime = Timer.getFPGATimestamp();
+    }
```

Comment:
shouldn't you use this function in a periodic?
if you think it should be another CR that's fine.

Reply:
wow that was fast